README.md

```diff
@@ -99,12 +101,22 @@ Get source code, run installer and follow on screen instructions:
 git clone https://github.com/AdnanHodzic/auto-cpufreq.git
 cd auto-cpufreq && sudo ./auto-cpufreq-installer
 ```
+### Update using auto-cpufreq-installer
+* The feature is available from version *1.9.8*. If your current version is below the same, Get source code, run installer and follow on screen instructions:
```

Comment:
I would remove everything after "* The feature is available from version *1.9.8*."

```
If your current version is below the same, Get source code, run installer and follow on screen instructions:
...
```

until line 113. Because it just extends the README for no reason and it might break things for some people, in case they are running on snap or AUR or just have auto-cpufreq under different path. If they want instructions just refer them to Install seciton.

Reply:
done